ompi/dpm/dpm.c

```diff
@@ -1974,6 +1975,46 @@ static void set_handler_default(int sig)
     sigaction(sig, &act, (struct sigaction *)0);
 }
 
+static char *find_prte(void)
```

Comment:
should the caller of find_prte being freed the returned string

Reply:
@hppritcha Are you saying that the pointer returned from `find_prte()` is not being freed?  It looks like it's being freed to me...?  Is there a case down in `start_dvm()` where the pointer is not freed?

Reply:
the problem is i don't like freeing a pointer returned by getenv.